Tests/SwiftCliCoreTests/PatternNormalizerTest.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+import XCTest
+import Nimble
+@testable import SwiftCliCore
+
+class PatternNormalizerTest: XCTestCase {
+
+    func testMultipleStitchStringIsLowerCased() throws {
+        let tryToNormalizeString = PatternNormalizer().makeAllLowercase(stitchesToLowercase: "P1 YO K1")
+        expect(tryToNormalizeString).to(equal("p1 yo k1"))
+    }
+
+    func testOneStitchStringIsLowerCased() throws {
+        let tryToNormalizeString = PatternNormalizer().makeAllLowercase(stitchesToLowercase: "K1")
+        expect(tryToNormalizeString).to(equal("k1"))
+    }
+
+    func testInconsistentlyCapitalizedStringIsLowerCased() throws {
+        let tryToNormalizeString = PatternNormalizer().makeAllLowercase(stitchesToLowercase: "k1 P1")
+        expect(tryToNormalizeString).to(equal("k1 p1"))
+    }
```

Comment:
For next pairing: playing off Hana's comment above, let's also talk about test naming next pairing session. Not a blocker though.  Merge away once you have addressed any additional comments on Friday 11/5!